docs/arch/scheduler.md

```diff
@@ -0,0 +1,28 @@
+# Scheduler
+
+## What?
+
+- A scheduler, its job is to schedule a job to an agent. This job is given by a controller
+- A scheduler does NOT create or launch agents.
+- A scheduler can work without any agents.
+- A scheduler knows the current state / capacity (memory, CPU) of each agent always.
```

Comment:
What do you mean by "A scheduler knows". Maybe we have to precise that he will spread jobs from the last information about CPU usage.